src/ui/Combobox/Combobox.ts

```diff
@@ -115,21 +109,33 @@ export class Combobox {
   }
 
   public updateAriaLive(text: string) {
-    this.options.searchInterface.ariaLive.updateText(text);
+    this.options.ariaLive.updateText(text);
+  }
+
+  public onScrollEndReached() {
+    this.values.saveFocusedValue();
+    this.options.scrollable &&
+      this.throttledRequest(
+        () => this.options.scrollable.requestMoreValues(),
+        () => this.values.restoreFocusedValue()
+      );
   }
 
-  private throttledTriggerNewRequest = throttle(this.triggerRequest, this.throttlingDelay, {
+  private throttledRequest = throttle(this.triggerRequest, this.throttlingDelay, {
     leading: true,
     trailing: true
   });
 
-  private async triggerRequest(terms: string) {
-    this.isThrottledRequestCancelled = false;
-    const response = await this.options.requestValues(terms);
+  private async triggerRequest(request: () => Promise<any>, callback?: Function) {
```

Comment:
Could the value resolved by the Promise be more strictly typed? Would it be the FacetSearch response?